packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -17,6 +17,98 @@ import {Application} from '@loopback/core';
  *
  * class MyApplication extends RepositoryMixin(Application) {}
  * ```
+ * 
+ * [function]
+ * 
+ * `repository(repo: Class<Repository<any>>): void {}`
+ * 
+ * [param]
```

Comment:
Could you please paste a screenshot showing how this long comment is rendered? Is the syntax `[function]` and `[param]` significant, i.e. will it create structured sections?

The current tsdoc comments looks like a long wall of text to me, it will be difficult for people to navigate around it.

Another thing to consider - is it possible to create a `#href` URL that will point directly to a particular mixin member, e.g. `getRepository` function? A possible solution is to use raw HTML and add `<a name="unique-slug">` before each new mixin member.